huaweicloud/services/acceptance/workspace/data_source_huaweicloud_workspace_app_shared_folders_test.go

```diff
@@ -0,0 +1,125 @@
+package workspace
+
+import (
+	"fmt"
+	"regexp"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance"
+)
+
+func TestAccDataSourceAppSharedFolders_basic(t *testing.T) {
+	var (
+		name = acceptance.RandomAccResourceName()
+
+		all = "data.huaweicloud_workspace_app_shared_folders.test"
+		dc  = acceptance.InitDataSourceCheck(all)
+
+		filterByStorageClaimId   = "data.huaweicloud_workspace_app_shared_folders.filter_by_storage_claim_id"
+		dcFilterByStorageClaimId = acceptance.InitDataSourceCheck(filterByStorageClaimId)
+
+		filterByPath   = "data.huaweicloud_workspace_app_shared_folders.filter_by_path"
+		dcFilterByPath = acceptance.InitDataSourceCheck(filterByPath)
+	)
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck: func() {
+			acceptance.TestAccPreCheck(t)
+			acceptance.TestAccPrecheckSfsFileSystemNames(t, 1)
+		},
+		ProviderFactories: acceptance.TestAccProviderFactories,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccDataSourceAppSharedFolders_basic(name),
+				Check: resource.ComposeTestCheckFunc(
+					dc.CheckResourceExists(),
+					resource.TestMatchResourceAttr(all, "shared_folders.#", regexp.MustCompile(`^[1-9]([0-9]*)?$`)),
+					dcFilterByStorageClaimId.CheckResourceExists(),
+					resource.TestCheckOutput("is_storage_claim_id_filter_useful", "true"),
+					dcFilterByPath.CheckResourceExists(),
+					resource.TestCheckOutput("is_path_filter_useful", "true"),
```

Comment:
```suggestion
					// Without any filter parameter.
					dc.CheckResourceExists(),
					resource.TestMatchResourceAttr(all, "shared_folders.#", regexp.MustCompile(`^[1-9]([0-9]*)?$`)),
					// Filter by 'storage_claim_id' parameter.
					dcFilterByStorageClaimId.CheckResourceExists(),
					resource.TestCheckOutput("is_storage_claim_id_filter_useful", "true"),
					// Filter by 'path' parameter.
					dcFilterByPath.CheckResourceExists(),
					resource.TestCheckOutput("is_path_filter_useful", "true"),
```
Using these comments can make each parts of acceptance test check clearly.

Reply:
done